spec/features/admin_edits_beer_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+feature 'Admin edits information for each individual beer page', %Q{
+    As an admin, I want to be able to edit
+    the information for all beers cause I’m in charge
+  } do
+
+#Acceptance Criteria
+#I must be signed in
+#I must be able to get here through the details page for a specific beer
+#I must fill in the required fields
+#I must be presented with errors if I fill out the form incorrectly
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.